main.js

```diff
@@ -0,0 +1,61 @@
+var Board = function (pegs, discs){
```

Comment:
stay consistent with your variable declarations - var is a pre-es6 way to declare a variable, and later on you switch over to let/const